client/js/controllers/user/profile.view.js

```diff
@@ -0,0 +1,18 @@
+angular
+  .module('app')
+  .controller('ProfileViewController', ['$scope', '$state', 'AuthService', 'Affiliation', 'User', '$stateParams', function ($scope, $state, AuthService, Affiliation, User, $stateParams) {
+
+    User.findById({id: $stateParams.userId, include: ['useraffiliation']}).$promise.then(function(u) {
```

Comment:
- Damit `include` funktioniert musst du es in ein `filter` Objekt schachteln (`filter: { include: ['useraffiliation'] }`).
- Die Relation `useraffiliation` ist für das Model `User` nicht definiert. Es gibt nur eine `affiliation`. Die funktioniert aber anscheinend nicht. Anscheinend war wohl auch mal vorgesehen dass ein user mehrere affiliations haben kann (n:m)?!. Brauchen wir aber nicht mehr, ist eine 1:n Beziehung. Von daher am besten einfach eine entsprechende Relation in `common/models/user.json` eintragen (https://docs.strongloop.com/display/public/LB/BelongsTo+relations)